packages/bpk-component-chip/src/BpkDismissibleChip.js

```diff
@@ -34,18 +35,29 @@ import {
   COMMON_DEFAULT_PROPS,
   type CommonProps,
 } from './commonTypes';
+import STYLES from './BpkSelectableChip.module.scss';
 
 export type Props = CommonProps;
 
+const getClassName = cssModules(STYLES);
+
 const BpkDismissibleChip = (props: Props) => {
-  const { ...rest } = props;
+  const { className, type, ...rest } = props;
 
   return (
     <BpkSelectableChip
       {...rest}
-      trailingAccessoryView={<CloseCircleIconSm />}
-      selected={false}
+      trailingAccessoryView={
+        <CloseCircleIconSm
+          className={getClassName(
```

Comment:
a very small thing here, could you maybe create a variable for this something like `iconClassNames` just because the class depends on the type so it's a bit cleaner if it's outside of this, what do you think?

Reply:
Yes, I agree. This way it will be clean and consistent with the previous code style. 😄